lib/browser/objects-registry.js

```diff
@@ -41,7 +41,12 @@ class ObjectsRegistry {
 
   // Get an object according to its ID.
   get (id) {
-    return this.storage[id].object
+    if (this.storage[id]) {
+      return this.storage[id].object
+    }
+
+    console.error(`Object id '${id}' does not exists in the object storage!`, this.storage)
+    return null
```

Comment:
I feel as though this would break more things than it solves.  Instead of an error being thrown we instead pretend to return a value (in this case null) which will almost certainly cause a different error to occur in the code?

That new error will be far less traceable to this issue, I think we should attempt to fix the root cause of the issue than just patch over it.

If @zcbenz is correct and it is a V8 bug the issue might simply be cleared out on the next Chromium bump (maybe) as our V8 version is linked to the Chromium version (I believe)